x-pack/plugins/infra/public/alerting/log_threshold/log_threshold_rule_type.ts

```diff
@@ -11,11 +11,11 @@ import { ObservabilityRuleTypeModel } from '../../../../observability/public';
 import {
   LOG_DOCUMENT_COUNT_ALERT_TYPE_ID,
```

Comment:
@ersin-erdal I would rename it to `LOG_DOCUMENT_COUNT_RULE_TYPE_ID` (but I would leave its value as it is)

Reply:
same as above